app/gradle/versionInfo.gradle

```diff
@@ -29,7 +29,7 @@ String getBuildDate() {
 
 String versionInfoFileContents() {
     kotlinLicenseHeader +
-    """ |package batect
+        """ |package batect
```

Comment:
Not sure what's happened here, but this formatting looks a little bit off...

Reply:
This seemed to resolved itself after I merged from upstream.